Games/types/Mafia/roles/cards/MeetingMafia.js

```diff
@@ -9,7 +9,7 @@ module.exports = class MeetingMafia extends Card {
       Mafia: {
         actionName: "Mafia Kill",
         states: ["Night"],
-        flags: ["group", "speech", "voting"],
+        flags: ["group", "speech", "voting", "multiActor"],
```

Comment:
multiActor is required to pass through this if check https://github.com/UltiMafia/Ultimafia/blob/81cb77acfe3404c4b127acc5b9b5a4f57a9e6c8e/Games/core/Meeting.js#L743

Reply:
Hold on, just to be clear: I believe that when a multiActor flag is applied, what that would mean is that for example if 3 mafia members vote to kill a player with a Bomb, all 3 die. Is this what you are intending? What are tests showing for you?

Reply:
@Golbolco cant run tests in my env for now. But checked manually your scenario and was ok
<img width="709" alt="Screenshot 2024-07-11 at 2 10 59 PM" src="https://github.com/UltiMafia/Ultimafia/assets/2591857/0c1f05b8-fe42-42da-bfe5-faffdccc51bf">
Without multActor, the current logic is that only the first Mafia to vote for the kill visits. 
With it, the code block goes through to put all actors in